src/main/kotlin/rmc/repository/rental/RentalRepositoryImpl.kt

```diff
@@ -2,16 +2,29 @@ package rmc.repository.rental
 
 import org.jetbrains.exposed.dao.id.EntityID
 import rmc.db.DatabaseFactory.dbQuery
+import rmc.db.dao.*
 import rmc.db.tables.UsersTable
 import rmc.db.tables.VehiclesTable
-import rmc.db.dao.*
 import rmc.dto.rental.CreateRentalDTO
 import rmc.dto.rental.RentalDTO
+import rmc.dto.rental.RentalId
```

Comment:
Wat is dit precies @AlexVDV116 ?

Reply:
RentalId is een [type alias](https://kotlinlang.org/docs/type-aliases.html) voor het type Int.
Deze worden in de DTO gedefineerd. Meer een manier om te verduidelijken welke type Int er wordt meegegeven.